src/pages/Login.tsx

```diff
@@ -17,36 +20,49 @@ interface UserCredentialLogin {
 
 interface Props {
   isLogin: boolean;
+  setIsLogin: React.Dispatch<React.SetStateAction<boolean>>;
 }
 
-const Login: React.FC<Props> = ({ isLogin }) => {
+const Login: React.FC<Props> = ({ isLogin, setIsLogin }) => {
   const navigate: (path: string) => void = useNavigate();
 
   useEffect(() => {
     if (isLogin === true) {
       navigate("/");
     }
-  }, []);
+  });
+
   const initalUser: UserCredentialLogin = {
     username: "",
     password: "",
   };
 
-  const [existingUser, setExistingUser] =
-    useState<UserCredentialLogin>(initalUser);
-  // if user does not exist in database
-  const [errorMsg, setErrorMsg] = useState<string>("");
+  const handleLogin = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    for (let i in data) {
```

Comment:
what is `data`?

Reply:
this my mock data, will change it to your db